### DISCUSSION TITLE
Are --load-in-8bit and --bf16 mutually exclusive?

### DISCUSSION THREAD

**Discussion by Slug-Cat** [oobabooga/text-generation-webui#84] [General]
If I add both, the server seems to run fine but I don't know whether they're actually doing anything.

**Comment by oobabooga** [comment]
Yes, `--bf16` is ignored if `--load-in-8bit` is specified:

https://github.com/oobabooga/text-generation-webui/blob/596732a98199de543f5c660c612b84adc18969a4/server.py#L139

**Reply by ack226** [reply]
Is one generally better to use than the other, assuming my GPU supports both?
